everyvoice/base_cli/interfaces.py

```diff
@@ -95,32 +95,6 @@ def train_base_command_interface(
 
 
 def inference_base_command_interface(
-    config_file: Path = typer.Argument(
-        ...,
-        exists=True,
-        dir_okay=False,
-        file_okay=True,
-        help="The path to your model configuration file.",
-        shell_complete=complete_path,
-    ),
     config_args: List[str] = typer.Option(None, "--config", "-c"),
-    accelerator: str = typer.Option(
```

Comment:
Hm, can you actually make an issue to refactor these fields (accelerator, devices, nodes) back into the helper and interface here? I think these should maybe be shared as opposed to buried in the FS2 code as they currently are.

Reply:
That's a good plan. The issue could include all these deleted interface declarations as documentation for whoever might implement this in the future.

Reply:
See #699